btcutil/psbt/finalizer.go

```diff
@@ -519,6 +519,9 @@ func finalizeTaprootInput(p *Packet, inIndex int) error {
 	switch {
 	// Key spend path.
 	case len(pInput.TaprootKeySpendSig) > 0:
+		if pInput.SighashType != txscript.SigHashDefault && len(pInput.TaprootKeySpendSig) == 64 {
```

Comment:
Can you wrap this to 80 characters? 

You can also use the `schnorr.SignatureSize` constant instead of `64 here.

Reply:
Finally, I think we also want a test here to ensure that the finalizer always adds the proper sighash flags if needed.